README.md

```diff
@@ -79,26 +79,38 @@ checking the output images.
 Options
 -------
 
-The ``@pytest.mark.mpl_image_compare`` marker can take an argument which is the
-RMS tolerance (which defaults to 2):
+Tolerance
+^^^^^^^^^
+
+The RMS tolerance for the image comparison (which defaults to 2) can be
```

Comment:
we have pushed the default tolerance in mpl down to 0, would that work with the other use-cases for this?